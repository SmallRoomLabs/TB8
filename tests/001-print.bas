10 print 1
20 print 11;22
30 print A
40 print A;B
50 print A;1
60 print A; 2
70 print A ;3
80 print A ; 4
90 print "A"
100 print "A";"B"
120 print "C";
130 print "D"
140 end
