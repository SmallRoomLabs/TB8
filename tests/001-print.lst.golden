>LIST
10 PRINT 1
20 PRINT 11 22
30 PRINT A
40 PRINT A ; B
50 PRINT A ; 1
60 PRINT A ; 2
70 PRINT A ; 3
80 PRINT A ; 4
90 PRINT "A"
100 PRINT "A" ; "B"
120 PRINT "C" ;
130 PRINT "D"
140 END

